convert2rhel/unit_tests/logger_test.py

```diff
@@ -64,10 +71,16 @@ def test_tools_opts_debug(tmpdir, read_std, is_py2):
         else:
             # this workaround is not working for py2 - passing
             pass
-    tool_opts.debug = False
+
+    global_tool_opts.debug = False
     logger.debug("debug entry 2: %s", "data")
     stdouterr_out, stdouterr_err = read_std()
     assert "debug entry 2: data" not in stdouterr_out
+    loggers = [logging.getLogger()] + list(logging.Logger.manager.loggerDict.values())
+    for logger in loggers:
+        handlers = getattr(logger, "handlers", [])
+        for handler in handlers:
+            logger.removeHandler(handler)
 
 
 def test_logger_custom_logger(tmpdir, caplog):
```

Comment:
```suggestion
def test_logger_custom_logger(tmpdir, caplog, clear_loggers):
```
This test also call the `convert2rhel.logger.setup_logger_handler()`.